templates/aws/nodes/terraform/pools/outputs.tf

```diff
@@ -1,8 +1,12 @@
+output "first_node_ip" {
+  value = aws_instance.node[0].public_ip
+}
+
 output "corral_node_pools" {
   value = {
     node = [for instance in aws_instance.node : {
       name = instance.tags.Name // unique name of node
-      user = "root" // ssh username
+      user = var.aws_ssh_user // ssh username
```

Comment:
Is there a reason you are changing this? I think this would break our custom cluster tests for airgap custom clusters.

Reply:
You can use other os than only the ones that have root as the default user. This is configurable, is the config not applied on your use case ?

Reply:
Root isn't necessarily default, it is when we do the `provisioner "remote-exec" {` that's the reason we're doing that. I just don't get why this needs to be changed.

Reply:
This adds non-root users capability for ssh, `local-exec` only adds keys for `root`

Reply:
This is to be able to run the corral scripts. I had issues with running scripts as non-root, I just don't quite remember why, but I remember having this issue with Jacob Payne was maintaining corral. This code adds keys for non-root users. We add this in the manifest, which our cluster-nodes has. I think this is the root of confusion 
https://github.com/rancherlabs/corral-packages/blob/5f0e0d10bcc4b2e84760817d77ad1432fcf564e0/templates/aws/cluster_nodes/manifest.yaml#L80-L86

Reply:
That's odd, you're still assuming `whoami` in this case in the original code will be `root`? Then if I understand as per your previous description you're adding access to an extra user to the same `root` account through ssh.

I could `tee` the echoed string to multiple files to add the public key to non-root users in the system and see how it works

Reply:
Yeah that looks like a mistake. If we fix the path, I think that would fix it. If you don't mind.